cantera/meta.yaml

```diff
@@ -13,6 +13,8 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # At the moment, the GH toolchain does not provide Fortran for Windows
```

Comment:
GH, meaning GitHub? These compilers are provided by Conda... I guess except on Windows where it's just a shim for Visual Studio. So I think the bigger issue is that there isn't a libre Fortran compiler for Windows at all...

Reply:
:+1: Removed "GH"